Projects/Modules/Documentarian.ModuleAuthor/Source/Public/Functions/Invoke-Pandoc.ps1

```diff
@@ -2,24 +2,73 @@
 # Licensed under the MIT License.
 
 function Invoke-Pandoc {
-
     param(
```

Comment:
Nit: This public function seems to be missing the cmdlet binding attribute and output type. Maybe we should explicitly define them?

```suggestion
    [CmdletBinding()]
    [OutputType([void])]
    param(
```